upstairs/src/live_repair.rs

```diff
@@ -3897,13 +3834,13 @@ pub mod repair_test {
 
         let eid = 1;
         ds.extent_limit[1] = Some(eid.try_into().unwrap());
-        let ddef = up.ddef.lock().await.get_def().unwrap();
-        let impacted_blocks = extent_to_impacted_blocks(&ddef, eid);
 
         // Upstairs "guest" work IDs.
         let gw_repair_id: u64 = gw.next_gw_id();
-        let repair_id = 1003;
-        let deps = vec![1001, 1002];
+        let (repair_ids, mut deps) = ds.get_repair_ids(eid);
+        assert!(deps.is_empty());
+        deps.push(1001);
+        deps.push(1002); // TODO: this is no longer testing something useful
```

Comment:
Same question here

Reply:
Same answer; this confirms that `create_and_enqueue_repair_io` uses the dependencies we pass in.